examples/fabric/byot/run.py

```diff
@@ -0,0 +1,80 @@
+import torch
+from torchmetrics.functional.classification.accuracy import accuracy
+from trainer import Trainer
+
+from lightning.pytorch import LightningModule
+
+
+class MNISTModule(LightningModule):
+    def __init__(self) -> None:
+        super().__init__()
+        self.model = torch.nn.Sequential(
+            torch.nn.Conv2d(
+                in_channels=1,
+                out_channels=16,
+                kernel_size=5,
+                stride=1,
+                padding=2,
+            ),
+            torch.nn.ReLU(),
+            torch.nn.MaxPool2d(kernel_size=2),
+            torch.nn.Conv2d(16, 32, 5, 1, 2),
+            torch.nn.ReLU(),
+            torch.nn.MaxPool2d(2),
+            torch.nn.Flatten(),
+            # fully connected layer, output 10 classes
+            torch.nn.Linear(32 * 7 * 7, 10),
+        )
+        self.loss_fn = torch.nn.CrossEntropyLoss()
+
+    def forward(self, x: torch.Tensor):
+        return self.model(x)
+
+    def training_step(self, batch, batch_idx: int):
+        x, y = batch
+
+        logits = self(x)
+
+        loss = self.loss_fn(logits, y)
+        accuracy_train = accuracy(logits.argmax(-1), y, num_classes=10, task="multiclass", top_k=1)
+
+        return {"loss": loss, "accuracy": accuracy_train}
```

Comment:
Why is the training step returning the accuracy?

Reply:
because there is no logging mechanism and you somehow need a way to return monitored values.

I could build a logging system, but that would either require modifications of the LM (definitely a no-go), or require some really ugly patching (also not ideal in an example)